### ISSUE TITLE
Rose stopped working

### ISSUE THREAD

**Issue by subjut** [ShyMemoriees/Rose#8]
Hi

Rose completely stopped working for me. I tried uninstalling the app, deleting the preference file, then iCleaning and reinstallting. It's installed and enabled, I also toggled that including resprings, and I have tried turning several options on and off. Nothing vibrates. I have done all of this several times.

Shortly before it stopped, I experienced a strange issue. Over night, my phone entered some weird jailbroken-but-not-really-jailbroken state. The home screen looked like it was not jailbroken. I could open cydia and do searches, but could not install anything or refresh sources. It always gave me an error and the option to respring, which I did, but I ended up in that same state again. I had to reboot and rejailbreak to get it working again, but since then Rose stopped working for me. As far as I can tell, all other tweaks still work.

Maybe you have some idea?

iPhone 7, iOS 12.4, unc0ver 3.6.2

**Comment by ShyMemoriees** [comment]
Heya

Have you installed the latest checkra1n version? Do other haptics work (system haptics, because there was a known bug for not working system haptics). And do you have bioprotect installed?

**Comment by subjut** [comment]
I am still running unc0ver. All other haptics seem to work (switching toggles for example). And I don't have bioprotect,

**Comment by ShyMemoriees** [comment]
Ohh you’re on iOS 12..  the Haptic issue with bioprotect was only on iOS 13 with checkra1n. What version of Rose are you running?

**Comment by subjut** [comment]
I'm on the newest Rose version, 3.3.

**Comment by ShyMemoriees** [comment]
This is weird, you’re the first one to have this issue. Have you already tried to disable all other tweaks or to downgrade to an older version?

**Comment by subjut** [comment]
I'll try to restore rootfs, rejailbreak and only installing Rose. I'll get back to you with the results.

**Comment by ShyMemoriees** [comment]
I will be waiting for your reply ^-^

**Comment by subjut** [comment]
OK! Rootfs restore and installing only Rose did work. I got the taptic engine feedback back. The settings were still there from before. After I restored a batchomatic deb file with all previous tweaks and settings. Then Rose did not work any more. But after disabling the taptic engine and the tweak itself in the Rose settings menu, then respringing, renenebling both and respringing again, Rose started working again. So all good! :-)

I think I tried disabling and enabling the tweak before, but I might not have done both the engine and the tweak at the same time. So the whole restore rootfs procedure might not have been necessary. Oops...

I'm just happy it works again! Thanks for the tweak.
